NitrokeyManager.cc

```diff
@@ -390,6 +390,69 @@ using nitrokey::misc::strcpyT;
         }
     }
 
+    uint8_t NitrokeyManager::get_pws_slot_count() {
+        if (device == nullptr) {
+            return 0;
+        } else {
+            return PWS_SLOT_COUNT;
+        }
+    }
+
+    size_t NitrokeyManager::get_pws_name_length() {
+        if (device == nullptr) {
+            return 0;
+        } else {
+            return PWS_SLOTNAME_LENGTH;
+        }
+    }
+
+    size_t NitrokeyManager::get_pws_login_length() {
+        if (device == nullptr) {
+            return 0;
+        } else {
+            return PWS_LOGINNAME_LENGTH;
+        }
+    }
+
+    size_t NitrokeyManager::get_pws_password_length() {
+        if (device == nullptr) {
+            return 0;
+        } else {
+            return PWS_PASSWORD_LENGTH;
+        }
+    }
+
+    uint8_t NitrokeyManager::get_totp_slot_count() {
+        if (device == nullptr) {
+            return 0;
+        } else {
+            return 15;
```

Comment:
I think listing constants in one place (e.g. header, like it is with `PWS_PASSWORD_LENGTH`), and just using them here would be best. 
Just an idea: potentially it might be easier to parse that with Python for offline testing too, instead of hardcoding the numbers there.

Reply:
Good point, fixed.